core/src/message/common.rs

```diff
@@ -24,7 +24,8 @@ use codec::{Decode, Encode};
 use core::ops::Deref;
 use scale_info::TypeInfo;
 
-/// Message.
+/// Structure for message between actors.
+/// Can be created from any type of messages and used to dispatch them.
```

Comment:
What's the purpose of the messages?

And for me writing "Structure" in docs seems weird.

Reply:
Rewrited it.